ci-conda.Dockerfile

```diff
@@ -37,7 +37,8 @@ EOF
 RUN <<EOF
 case "${LINUX_VER}" in
   "ubuntu"*)
-    apt-get update && apt-get install -y wget
+    i=0; until apt-get update -y; do ((++i >= 5)) && break; sleep 10; done
```

Comment:
Why are some of these using this `until` loop and others using `rapids-retry`?

If it was to customize the number of retries or sleep time, both can be controlled with shell variables.

```shell
RAPIDS_RETRY_MAX=5 \
RAPIDS_RETRY_SLEEP=10 \
  rapids-retry apt-get update -y
```

source: https://github.com/rapidsai/gha-tools/blob/f25369682c8062d497803e72c4dd5815892d1fad/tools/rapids-retry#L49-L51

Reply:
This line is just before we install `wget` which we then use to install `gha-tools`, so for this particular instance we need to loop manually